nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -158,15 +158,15 @@ contract Issuer is Upgradeable {
     }
 
     function verifyState(address _testTarget) public onlyOwner {
-        require(address(delegateGet(_testTarget, "token()")) == address(token));
-        require(uint256(delegateGet(_testTarget, "miningCoefficient()")) == miningCoefficient);
-        require(uint256(delegateGet(_testTarget, "lockedPeriodsCoefficient()")) == lockedPeriodsCoefficient);
+        require(address(uint160(delegateGet(_testTarget, "token()"))) == address(token));
+        require(delegateGet(_testTarget, "miningCoefficient()") == miningCoefficient);
+        require(delegateGet(_testTarget, "lockedPeriodsCoefficient()") == lockedPeriodsCoefficient);
         require(uint32(delegateGet(_testTarget, "secondsPerPeriod()")) == secondsPerPeriod);
         require(uint16(delegateGet(_testTarget, "rewardedPeriods()")) == rewardedPeriods);
         require(uint16(delegateGet(_testTarget, "lastMintedPeriod()")) == lastMintedPeriod);
-        require(uint256(delegateGet(_testTarget, "currentSupply1()")) == currentSupply1);
-        require(uint256(delegateGet(_testTarget, "currentSupply2()")) == currentSupply2);
-        require(uint256(delegateGet(_testTarget, "totalSupply()")) == totalSupply);
+        require(delegateGet(_testTarget, "currentSupply1()") == currentSupply1);
```

Comment:
Is this happening because you don't need to cast to `uint256` anymore in 0.5?

Reply:
casting was restricted in 0.5, because of different padding for different types (bytes and uint). With this changings method `delegateGet` (in `Upgreadable`) have padding like `uint256` and not how `bytes32`, so changed return type and using of this method